src/pages/spawn/SpawnHost.tsx

```diff
@@ -1,3 +1,78 @@
 import React from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { H2 } from "@leafygreen-ui/typography";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import { pollInterval } from "constants/index";
+import { useBannerDispatchContext } from "context/banners";
+import { MyHostsQuery, MyHostsQueryVariables } from "gql/generated/types";
+import { GET_MY_HOSTS } from "gql/queries";
+import { SpawnHostButton } from "pages/spawn/spawnHost/index";
+import { useNetworkStatus } from "hooks";
+import { HostStatus } from "types/host";
 
-export const SpawnHost = () => <div>spawn host page</div>;
+export const SpawnHost = () => {
+  const dispatchBanner = useBannerDispatchContext();
+
+  const { data, loading, startPolling, stopPolling } = useQuery<
+    MyHostsQuery,
+    MyHostsQueryVariables
+  >(GET_MY_HOSTS, {
+    pollInterval,
+    onError: (e) => {
+      dispatchBanner.errorBanner(
+        `There was an error loading the patch: ${e.message}`
+      );
+      console.log(e);
```

Comment:
was this intentional?

Reply:
yeah this is still a wip which will be addressed in a follow up pr